frontends/web/src/components/copy/Copy.tsx

```diff
@@ -35,15 +35,12 @@ interface State {
 }
 
 class CopyableInput extends Component<Props, State> {
-    private inputField!: HTMLInputElement;
-
-    constructor(props: Props) {
-        super(props);
-        this.state = {
-            success: false,
-        };
+    public readonly state: State = {
+        success: false,
     }
 
+    private textArea!: HTMLTextAreaElement;
```

Comment:
What's the difference? I mean, why is textarea better than input here. Would've been nice if the commit msg explained it.

Reply:
The copy component is already using a textarea instead of an input element, this was changed in https://github.com/digitalbitbox/bitbox-wallet-app/commit/ce2fdd7ea4064b15fdb6dd6b93ffdd4b9e8206a5#diff-b5a510b0817e11affa3032785dc7cf269fce7aee513f5ea3f73a2f9a11a118fb
in order to support multiple lines, but all the variable names and types are still using input.

Updated the commit message.